README.md

```diff
@@ -77,6 +77,11 @@ $ git lfs track "*.psd"
 more about this pattern syntax
 [here](https://git-scm.com/docs/gitattributes)).
 
+**Note**: the quotations surrounding the pattern are important. Otherwise
```

Comment:
It may be good to match the existing style of the "you-should-probably-know-about-this" as we have elsewhere in the README.md, such as writing it instead with a blockquote and italics, as:

```md
> *Note*: the quotations surrounding the pattern are important [...]
```

Reply:
👍